webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  entry: {
+    rollbar: './src/bundles/rollbar.js',
+    'rollbar.nojson': './src/bundles/rollbar.nojson.js',
+    'rollbar.umd': './src/bundles/rollbar.umd.js'
+  },
+  output: {
+    library: 'Rollbar',
+    path: './dist/bundles/',
```

Comment:
Does this mean that we will no longer be placing the distributed, (minified, etc.) files into `dist/`? I think we should keep them there.